### PR TITLE
Added support for njs module

### PR DESCRIPTION
**Pull request by ronanj**

### CODE REVIEW

File: Formula/njs-nginx-module.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  homepage "https://github.com/nginx/njs"
+  url "https://github.com/nginx/njs/archive/master.zip"
+  version "1.0"
+  #Note that the njs github repo is an automatic sync from a mercurial repo
+  sha256 "3077b2c79a79537007213ca1f23ce5f94f51520bfe267f9b7f8127c8b823b722"
+
+  bottle :unneeded
+
+  def install
+    (share+"njs-nginx-module").install Dir["*"]
```
Comment (denji) [inline Formula/njs-nginx-module.rb]: `(share+"njs-nginx-module").install Dir["nginx/*"]`

File: Formula/nginx-full.rb [hunk 2]
```diff
@@ -246,9 +247,14 @@ def install
...
 
+    # Njs module configuration path needs to be appended with "/nginx" compared to other modules
+    if build.with?("njs-module")
+      args << "--add-module=#{HOMEBREW_PREFIX}/share/njs-nginx-module/nginx"
+    end
+
     # Third Party Modules
     self.class.third_party_modules.each_key do |name|
-      if build.with? "#{name}-module"
+      if build.with? "#{name}-module" && name != "njs"
```
Comment (denji) [inline Formula/nginx-full.rb]: This is not necessary, it is necessary to correct in `njs-nginx-module` set subdir.

File: Formula/njs-nginx-module.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class NjsNginxModule < Formula
+  desc "Nginx NJS module"
+  homepage "https://github.com/nginx/njs"
+  url "https://github.com/nginx/njs/archive/master.zip"
```
Comment (denji) [inline Formula/njs-nginx-module.rb]:
```ruby
  url "https://github.com/nginx/njs/tarball/master/nginx-njs-c4a5f2b.tar.gz"
  sha256 "213d96e839bbc6e979708f42677aabcb723d0eaed27fa9aef198bb9e024e51d9"
```

Comment (ronanj) [inline Formula/njs-nginx-module.rb]: Thanks. This is updated

File: Formula/nginx-full.rb (hunk 2 above)
Comment (ronanj) [inline Formula/nginx-full.rb]: I could not it to work - combined with the `(share+"njs-nginx-module").install Dir["nginx/*"]`. The main reason is that the njs module does see the nginx module as a configuration only, and the rest of the code is in another folder ("../nxt" and "../njs").

**Pull request closed without merging**